conftest.py

```diff
@@ -451,24 +451,39 @@ def dask_env_variables():
     return {k: v for k, v in os.environ.items() if k.startswith("DASK_")}
 
 
-@pytest.fixture(scope="module")
-def small_cluster(request, dask_env_variables, gitlab_cluster_tags):
-    # Extract `backend_options` for cluster from `backend_options` markers
-    backend_options = merge(
-        m.kwargs for m in request.node.iter_markers(name="backend_options")
-    )
-    backend_options["send_prometheus_metrics"] = True
+@pytest.fixture(scope="session")
+def cluster_kwargs() -> dict:
+    base_dir = os.path.dirname(__file__)
+    base_fname = os.path.join(base_dir, "cluster_kwargs.yaml")
+    with open(base_fname) as fh:
+        config = yaml.safe_load(fh)
+
+    override_fname = os.getenv("CLUSTER_KWARGS")
+    if override_fname:
+        override_fname = os.path.join(base_dir, override_fname)
+        with open(override_fname) as fh:
+            override_config = yaml.safe_load(fh)
+        dask.config.update(config, override_config)
+
+    default = config.pop("default")
+    config = {k: dask.config.merge(default, v) for k, v in config.items()}
 
+    # For forensic analysis
+    output_fname = os.path.join(base_dir, "cluster_kwargs.merged.yaml")
+    with open(output_fname, "w") as fh:
+        yaml.dump(config, fh)
+
+    return config
+
+
+@pytest.fixture(scope="module")
+def small_cluster(request, dask_env_variables, cluster_kwargs, gitlab_cluster_tags):
     module = os.path.basename(request.fspath).split(".")[0]
     with Cluster(
         name=f"{module}-{uuid.uuid4().hex[:8]}",
-        n_workers=10,
-        worker_vm_types=["m6i.large"],  # 2CPU, 8GiB
-        scheduler_vm_types=["m6i.xlarge"],
-        backend_options=backend_options,
-        package_sync=True,
         environ=dask_env_variables,
         tags=gitlab_cluster_tags,
```

Comment:
Minor concern with this is that our benchmarks would not be able to pick up a difference here. It'd be nice if these kwargs would be hashed and/or stored in the DB somewhere

Reply:
The db stores a link to the stdout/stderr of the run, and in that you have the dump of the kwargs